Travis builds fail because git.apache.org is down

## Expected Behavior
Travis should load all module requirements.
## Actual Behavior
Building the provider fails
```
go: git.apache.org/thrift.git@v0.0.0-20180902110319-2566ecd5d999: git fetch -f origin refs/heads/*:refs/heads/* refs/tags/*:refs/tags/* in /home/travis/gopath/pkg/mod/cache/vcs/83dba939f95a790e497d565fc4418400145a1a514f955fa052f662d56e920c3e: exit status 128:

	fatal: unable to access 'https://git.apache.org/thrift.git/': Failed to connect to git.apache.org port 443: Connection timed out

go: error loading module requirements

Makefile:29: recipe for target 'build' failed

make: *** [build] Error 1

The command "make build" exited with 2.

```

## Steps to Reproduce the Problem
Attempt to build with Travis

## Additional info
People have reported the issue on the terraform repository
https://github.com/hashicorp/terraform/issues/22664

A suggested fix is to set `GOPROXY=https://proxy.golang.org` or use Go 1.13

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,7 @@ addons:
     # needed for the snap pipe:
     - snapd
 go:
-- 1.12.x
+- 1.13.x
 install:
 # needed for the snap pipe:
 - sudo snap install snapcraft --classic

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ This provider plugin is maintained by the Terraform team at [Articulate](https:/
 ## Requirements
 
 - [Terraform](https://www.terraform.io/downloads.html) 0.12.x
-- [Go](https://golang.org/doc/install) 1.12 (to build the provider plugin)
+- [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
 
 ## Demo
 

